### DISCUSSION TITLE
Question about _filter parameter

### DISCUSSION THREAD

**Discussion by m-onyshchuk** [LinuxForHealth/connect#352] [Q&A] [answered]
Hello everybody!
I am trying to use _filter parameter ( http://hl7.org/fhir/search_filter.html ) for building of complex patient search.
My requests 
```
/fhir-server/api/v4/Patient?family=Smith
/fhir-server/api/v4/Patient?given=John
```
worked fine. But next: 
```
/fhir-server/api/v4/Patient?_filter=(family eq 'Smith' or given eq 'John')
```
returned 
```
Search parameter '_filter' for resource type 'Patient' was not found.
```
Due to https://linuxforhealth.github.io/FHIR/guides/FHIRSearchConfiguration/#13-filtering 
I have placed next `fhirServer/resources` property group:
```
"resources": {
  "open": false,
  "Patient": {
    "searchParameters": {
      "family": "http://ibm.com/fhir/SearchParameter/Patient-family",
      "given": "http://ibm.com/fhir/SearchParameter/Patient-given"
    }
  }  
}

```
to `/opt/ol/wlp/usr/servers/defaultServer/config/default/fhir-server-config.json`
but had got mistake in logs:
```
Error while computing search parameters for tenant default; returning empty
```   
What I can to do to run complex search like `_filter=(family eq 'Smith' or given eq 'John')` ?

**Comment by lmsurpre** [comment, marked as answer]
`_filter` is not supported on the LinuxForHealth FHIR server.  there's an issue to track it, but it would be a big lift to implement it and we have no plans for doing that:  https://github.com/LinuxForHealth/FHIR/issues/3098

the best place to look for what is/isn't supported is https://linuxforhealth.github.io/FHIR/Conformance
